source/includes/kubernetes_extension/_k8_charts.md

```diff
@@ -258,10 +242,13 @@ Required | &nbsp;
 
 Attributes | &nbsp;
 ------- | -----------
-`relationships` <br/>*objection* | The relationship information for the chart.
-`relationships.latestChartVersion` <br/>*object* | The lastest version chart information.
-`relationships.latestChartVersion.data.urls` <br/>*string* | The URLs for the package to download.
-`insecureServer`<br/>*boolean* | Specify if to access the chart we need an unsecure connection.
-`passedValues` <br/>*string* | The values to use to install the chart. It must be a valid Yaml format.
-`namespace` <br/>*map* | The namespace in which the chart will be installed.
-`releaseName` <br/>*string* | The name under which the chart will be known.
+| Attributes                      | &nbsp;                                                                           |
+|---------------------------------|----------------------------------------------------------------------------------|
+| `namespace` <br/>_string_       | The namespace in which the chart will be installed.                              |
+| `releaseName` <br/>_string_     | The desired release name.                                                        |
+| `name` <br/>_string_            | The chart name.                                                                  |
+| `version` <br/>_string_         | The chart version to be installed. Defaults to the latest version if left blank. |
+| `values` <br/>_string_          | The values to use to install the chart. It must be a valid Yaml format.          |
```

Comment:
is it mandatory?

Reply:
Nice catch, fixed here https://github.com/cloudops/cloudmc-api-docs/pull/261/commits/498b4777ba7f890f506f25c754059b8484c23de2#diff-3cbb097edb0517bba241163d615649f31d6e6cf393ad6356b544586e5d1ff8e8R232